algorithms/linfa-elasticnet/examples/elasticnet.rs

```diff
@@ -7,7 +7,7 @@ fn main() -> Result<()> {
 
     // train pure LASSO model with 0.1 penalty
     let model = ElasticNet::params()
-        .penalty(0.1)
+        .penalty(0.3)
```

Comment:
Should the comment be updated too?

Reply:
should be `0.1` here, I was playing around with the example